easylist/easylist_adservers.txt

```diff
@@ -1365,6 +1365,7 @@
 ||amtminjo.com^
 ||amtracking01.com^
 ||amuletlatemanagement.com^
+||analytics.new-porn.club^
```

Comment:
In the tracking list.... https://github.com/easylist/easylist/blob/master/easyprivacy/easyprivacy_specific.txt 

https://mypdns.org/my-privacy-dns/porn-records/-/issues/1772